solana/system_program.py

```diff
@@ -198,17 +198,17 @@ class AssignWithSeedParams(NamedTuple):
 
 
 SYSTEM_INSTRUCTION_LAYOUTS: List[InstructionLayout] = [
-    InstructionLayout(idx=_CREATE, fmt="<Iqq32s"),
-    InstructionLayout(idx=_ASSIGN, fmt="<I32s"),
-    InstructionLayout(idx=_TRANSFER, fmt="<Iq"),
-    InstructionLayout(idx=_CREATE_WITH_SEED, fmt=""),
-    InstructionLayout(idx=_ADVANCE_NONCE_ACCOUNT, fmt="<I"),
-    InstructionLayout(idx=_WITHDRAW_NONCE_ACCOUNT, fmt="<Iq"),
-    InstructionLayout(idx=_INITIALZE_NONCE_ACCOUNT, fmt="<I32s"),
-    InstructionLayout(idx=_AUTHORIZE_NONCE_ACCOUNT, fmt="<I32s"),
-    InstructionLayout(idx=_ALLOCATE, fmt="<I32s"),
-    InstructionLayout(idx=_ALLOCATE_WITH_SEED, fmt=""),
-    InstructionLayout(idx=_ASSIGN_WITH_SEED, fmt=""),
+    InstructionLayout(idx=CREATE_IDX, fmt="<Iqq32s"),
```

Comment:
I was thinking if we should stop requiring `<I` from the formats, and just always prepend the `<I` prefix in `encode_data`. What do you think?

Reply:
Sounds good.